.clang-format

```diff
@@ -0,0 +1,4 @@
+BasedOnStyle: LLVM
+IndentWidth: 4
+SortIncludes: false
```

Comment:
This is necessary because the code is structured on include order, and I didn't want to spend an entire afternoon just rebuilding a dependency graph.